concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-58	58	the field	field	212	FIELD	
+Zhang-2019-308-59	59	the fire	fire	221	FIRE	
+Zhang-2019-308-60	60	the firewood	firewood	10	FIREWOOD	
+Zhang-2019-308-61	61	five	five	493	FIVE	
+Zhang-2019-308-62	62	to float	float	1574	FLOAT	
+Zhang-2019-308-63	63	the fly	fly (insect)	1504	FLY (INSECT)	
+Zhang-2019-308-64	64	be fresh	fresh	243	FRESH	
+Zhang-2019-308-65	65	the friend	friend	1325	FRIEND	
+Zhang-2019-308-66	66	the fur garment	fur garment			
+Zhang-2019-308-67	67	the garment	garment	1895	CLOTHES	
+Zhang-2019-308-68	68	the grandchild, nephew	grandchild, nephew	1617	GRANDCHILD	
+Zhang-2019-308-69	69	the grandmother	grandmother	1496	GRANDMOTHER	
+Zhang-2019-308-70	70	the hair	hair	1040	HAIR	
+Zhang-2019-308-71	71	the hat	hat	771	HAT	
+Zhang-2019-308-72	72	the head	head	1256	HEAD	
+Zhang-2019-308-73	73	the heart	heart	1223	HEART	
+Zhang-2019-308-74	74	the hide	hide	3326	LEATHER OR HIDE	
+Zhang-2019-308-75	75	the hole	hole	1667	HOLE	
+Zhang-2019-308-76	76	the honey	honey	942	HONEY	
+Zhang-2019-308-77	77	the horn	horn	1393	HORN (ANATOMY)	
+Zhang-2019-308-78	78	the horse	horse	615	HORSE	
+Zhang-2019-308-79	79	hundred	hundred	1634	HUNDRED	
+Zhang-2019-308-80	80	the husband	husband	1200	HUSBAND	
+Zhang-2019-308-81	81	the ice	ice	617	ICE	
+Zhang-2019-308-82	82	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-83	83	the internal organs	internal organs	1334	GUTS	
+Zhang-2019-308-84	84	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-85	85	the jaw, chin	jaw, chin	798	JAW	
+Zhang-2019-308-86	86	the joint	joint	999	JOINT	
+Zhang-2019-308-87	87	be late	late	425	BE LATE	
+Zhang-2019-308-88	88	the louse	louse	1392	LOUSE	
+Zhang-2019-308-89	89	the milk, nipple	milk, nipple	2129	BREAST OR MILK	
+Zhang-2019-308-90	90	the millet	millet	931	MILLET	
+Zhang-2019-308-91	91	the millet or rice grains	millet or rice grains	931	MILLET	
+Zhang-2019-308-92	92	the mother's brother	mother's brother	2692	MATERNAL UNCLE (MOTHER'S BROTHER)	
+Zhang-2019-308-93	93	the mushroom	mushroom	641	MUSHROOM	
+Zhang-2019-308-94	94	the nail	nail	502	NAIL (TOOL)	
+Zhang-2019-308-95	95	the name	name	1405	NAME	
+Zhang-2019-308-96	96	be near	near	1942	NEAR	
+Zhang-2019-308-97	97	the needle	needle	1382	NEEDLE (FOR SEWING)	
+Zhang-2019-308-98	98	the night	night	1233	NIGHT	
+Zhang-2019-308-99	99	nine	nine	1483	NINE	
+Zhang-2019-308-100	100	not to be	not be	1240	NOT	
+Zhang-2019-308-101	101	one	one	1493	ONE	
+Zhang-2019-308-102	102	the onion	onion	2366	ONION	
+Zhang-2019-308-103	103	the owl	owl	735	OWL	
+Zhang-2019-308-104	104	the ox	ox	1169	OX	
+Zhang-2019-308-105	105	to peel off	peel off	275	PEEL	
+Zhang-2019-308-106	106	the pig	pig	1337	PIG	
+Zhang-2019-308-107	107	the pillow	pillow	1044	PILLOW	
+Zhang-2019-308-108	108	the plant root	plant root	670	ROOT	
+Zhang-2019-308-109	109	the pliers	pliers			
+Zhang-2019-308-110	110	the poison	poison	655	POISON	
+Zhang-2019-308-111	111	the pulse, beans	pulse, beans	832	BEAN	
```

Comment:
Wikipedia says pulses can refer to various kinds of seeds, including beans, peas, lentils, etc. So I think this case calls for a broader concept set.